services/install-calibre-web.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/calibre-web
+
+  # create yml(s)
+  cat << EOF > /srv/calibre-web/calibre-web.yml
+version: "2.1"
+services:
+  calibre-web:
+    image: linuxserver/calibre-web
+    environment:
+      - PUID=${PUID}
+      - PGID=${PGID}
+      - TZ=${TZ}
+      - DOCKER_MODS=linuxserver/calibre-web:calibre
+    volumes:
+      - "/srv/calibre-web:/root/.calibre-web"
+      - "/srv/calibre-web:/books:/books"
+    ports:
+      - 8083:8083
+    restart: unless-stopped
+EOF
+
+  # create .env with default values
+  cat << EOF > /srv/calibre-web/.env
+PUID=1000
+PGID=1000
+TZ=Europe/London
+PASSWORD=PASSWORD
+DB_TYPE=sqlite
+DB_NAME=DB_NAME
+DB_HOSTNAME=DB_HOSTNAME
+DB_USERNAME=DB_USERNAME
+DB_PASSWORD=DB_PASSWORD
```

Comment:
I don't think these environment variables are used anywhere.
Could probably be removed.